README.md

```diff
@@ -30,32 +45,56 @@ height="75px">
 
 ## Screenshots
 
-![](./packages/web/images/screenshots/screenshot-230602-a.png)
-
+![](./packages/web/images/screenshots/screenshot-2023-10-19.png)
 
 ## Project Setup
 
+### Stack
+- [NodeJS](https://nodejs.org/) (v18)
+- Back: [Express](http://expressjs.com/) (v4)
+- Front: [VueJS](https://vuejs.org/) (v3)
```

Comment:
Perhaps we could mention the DSFR package here, just make it more visible from the start ? 

```md
- Design system: [DSFR](https://www.systeme-de-design.gouv.fr/)
```